en/config_vtol/vtol_quad_configuration.md

```diff
@@ -117,15 +115,15 @@ Make sure the VTOL is in a stable hover before you start the transition.
 Start your transition.
 It should transition within 50 – 100 meters.
 If it doesn’t or it isn’t flying in a stable fashion abort the transition (see below) and land or hover back to the start position and land.
-Try increasing the [transition throttle](#transition_throttle) (`VT_F_TRANS_THR`) value.
+Try increasing the [transition throttle](#transition-throttle) (`VT_F_TRANS_THR`) value.
 Also consider reducing the transition duration (`VT_F_TRANS_DUR`) if you are not using an airspeed sensor.
 If you are using an airspeed sensor consider lowering the transition airspeed but stay well above the stall speed.
 
 As soon as you notice the transition happen be ready to handle height loss which may include throttling up quickly.
 
-:::caution
+:::warning

```

Comment:
FYI, caution not supported in vuepress by default. Decided to just use warning everywhere.